ssh-keygentest.py

```diff
@@ -0,0 +1,14 @@
+import unittest
+from testutils import system
+ 
+class Firetest(unittest.TestCase):
+    def test_cat(self):
+        out,err,retcode=system('ssh-keygen -t rsa -f ~/.ssh/id_rsa -q -P ""')
+        out,err,retcode=system('cat ~/.ssh/id_rsa.pub')
```

Comment:
Add minimal spaces for line6 and 7.
Like `out, err, retcode = system('cat ~/.ssh/id_rsa.pub')` instead of `out,err,retcode=system('cat ~/.ssh/id_rsa.pub')`